cmd/gsw/axtHelper.go

```diff
@@ -33,8 +34,16 @@ func convertAxt(axtFile, format, targetFa, output string) {
 }
 
 func goChannelAxtVcf(axtFile string) <-chan *vcf.Vcf {
+	file := fileio.EasyOpen(axtFile)
+	var wg sync.WaitGroup
 	axtChannel := make(chan axt.Axt, 2408)
-	go axt.ReadToChan(axtFile, axtChannel)
+	wg.Add(1)
+	go axt.ReadToChan(file, axtChannel, &wg)
+
+	go func() {
+		wg.Wait()
```

Comment:
I think this might be a second example of the waiting go routine not closing the file.